build.sh

```diff
@@ -41,9 +41,9 @@ check go                       "needs go from http://golang.org"
 check $GOPATH/bin/gometalinter "needs gometalinter from https://github.com/alecthomas/gometalinter"
 check $GOPATH/bin/go-bindata   "needs go-bindata from https://github.com/jteeuwen/go-bindata"
 
-perform builds     go install . ./operations
-perform linters    gometalinter --fast -e "credentials,HIGH,LOW"
 perform generators go generate . ./operations
+perform linters    gometalinter --fast --aggregate
+perform builds     go install . ./operations
```

Comment:
I have just asked in [gitter](https://gitter.im/alecthomas/Lobby) of gometalinter that `Should we run gometalinter before or after go build or go install?` and they said 
```
best after
some linters operate on the compiled artifacts
```
So I think we should run gometalinter again after `go install`  or only `go install`